src/plugins/intel_gpu/src/graph/include/eltwise_inst.h

```diff
@@ -35,6 +35,9 @@ struct typed_program_node<eltwise> : public typed_program_node_base<eltwise> {
         return std::make_shared<EltwiseFuseParams>(typed_desc());
     }
     std::vector<size_t> get_shape_infer_dependencies() const override { return {}; }
+
+    bool need_input_tensors_size_align() const;
+    std::optional<size_t> find_eltwise_const_dep_idx() const;
```

Comment:
this is only used in primitive_fusing. Is it worth to have this function as a permanent member function?

Reply:
Removed from member.